pkg/vrraw/vrraw_test.go

```diff
@@ -0,0 +1,159 @@
+package vrraw_test
+
+import (
+	"github.com/suyashkumar/dicom/pkg/vrraw"
+	"testing"
+)
+
+// Tests that the VR consts are what we expect
+func TestVRRawValues(t *testing.T) {
+	testCases := []struct {
+		VR       string
+		Expected string
+	}{
+		{
+			VR:       vrraw.ApplicationEntity,
+			Expected: "AE",
+		},
+		{
+			VR:       vrraw.AgeString,
+			Expected: "AS",
+		},
+		{
+			VR:       vrraw.AttributeTag,
+			Expected: "AT",
+		},
+		{
+			VR:       vrraw.CodeString,
+			Expected: "CS",
+		},
+		{
+			VR:       vrraw.Date,
+			Expected: "DA",
+		},
+		{
+			VR:       vrraw.DecimalString,
+			Expected: "DS",
+		},
+		{
+			VR:       vrraw.DateTime,
+			Expected: "DT",
+		},
+		{
+			VR:       vrraw.FloatingPointSingle,
+			Expected: "FL",
+		},
+		{
+			VR:       vrraw.FloatingPointDouble,
+			Expected: "FD",
+		},
+		{
+			VR:       vrraw.IntegerString,
+			Expected: "IS",
+		},
+		{
+			VR:       vrraw.LongString,
+			Expected: "LO",
+		},
+		{
+			VR:       vrraw.LongText,
+			Expected: "LT",
+		},
+		{
+			VR:       vrraw.OtherByte,
+			Expected: "OB",
+		},
+		{
+			VR:       vrraw.OtherDouble,
+			Expected: "OD",
+		},
+		{
+			VR:       vrraw.OtherFloat,
+			Expected: "OF",
+		},
+		{
+			VR:       vrraw.OtherLong,
+			Expected: "OL",
+		},
+		{
+			VR:       vrraw.OtherVeryLong,
+			Expected: "OV",
+		},
+		{
+			VR:       vrraw.OtherWord,
+			Expected: "OW",
+		},
+		{
+			VR:       vrraw.PersonName,
+			Expected: "PN",
+		},
+		{
+			VR:       vrraw.ShortString,
+			Expected: "SH",
+		},
+		{
+			VR:       vrraw.SignedLong,
+			Expected: "SL",
+		},
+		{
+			VR:       vrraw.Sequence,
+			Expected: "SQ",
+		},
+		{
+			VR:       vrraw.SignedShort,
+			Expected: "SS",
+		},
+		{
+			VR:       vrraw.ShortText,
+			Expected: "ST",
+		},
+		{
+			VR:       vrraw.SignedVeryLong,
+			Expected: "SV",
+		},
+		{
+			VR:       vrraw.Time,
+			Expected: "TM",
+		},
+		{
+			VR:       vrraw.UnlimitedCharacters,
+			Expected: "UC",
+		},
+		{
+			VR:       vrraw.UniqueIdentifier,
+			Expected: "UI",
+		},
+		{
+			VR:       vrraw.UniversalResourceIdentifier,
+			Expected: "UR",
+		},
+		{
+			VR:       vrraw.UnsignedLong,
+			Expected: "UL",
+		},
+		{
+			VR:       vrraw.Unknown,
+			Expected: "UN",
+		},
+		{
+			VR:       vrraw.UnsignedShort,
+			Expected: "US",
+		},
+		{
+			VR:       vrraw.UnlimitedText,
+			Expected: "UT",
+		},
+		{
+			VR:       vrraw.UnsignedVeryLong,
+			Expected: "UV",
+		},
+	}
+
+	for _, thisCase := range testCases {
+		t.Run(thisCase.Expected, func(t *testing.T) {
+			if thisCase.Expected != thisCase.VR {
+				t.Errorf("expected %v, got %v", thisCase.Expected, thisCase.VR)
```

Comment:
nit: you can consider a shorter variable name for `thisCase` if you prefer (see https://github.com/golang/go/wiki/CodeReviewComments#variable-names), since it is used in a small scope within a for loop (commonly you'll see `tc` here as an abbreviation for "test case" or "this case").

This sort of thing is definitely one of the quirks of Go that it takes getting used to seeing, but I've found in practice it doesn't hurt readability when used in the right scopes. 

```suggestion
	for _, tc := range testCases {
		t.Run(tc.Expected, func(t *testing.T) {
			if tc.Expected != tc.VR {
				t.Errorf("expected %v, got %v", tc.Expected, tc.VR)
```

Reply:
No problem.

Reply:
Since the test were removed, no need to fix, but I will update my other PRs.